src/modules/auth/auth_repository.ts

```diff
@@ -80,6 +81,14 @@ export class AuthRepository {
   public updatePassword = (id: string, password: string) =>
     this.Users().where('id', id).update({ password })
 
-  public updateLastLoginAt = (id: string) =>
-    this.Users().where('id', id).update({ last_login_at: new Date() })
+  public updateAfterSignIn = (user: AuthEntity.User) => {
+    let payload: AuthEntity.User
+
+    payload.last_login_at = new Date()
```

Comment:
eh jadi kepikiran. Ini `new Date()` defaultnya bikin waktu dengan timezone apa ya? emang pasti ke UTC ya?

Reply:
maaf, saya belum terlalu paham. Kalau new Date() disini itu, valuenya datetime (2022-08-19 15:27) atau DATE (2022-08-19) ya?

Reply:
jadi iseng nyoba
![image](https://user-images.githubusercontent.com/32012588/185580722-ba09525c-3d90-40d0-9601-1350c896148b.png)

convert to ISO format
![image](https://user-images.githubusercontent.com/32012588/185581486-b05b7cc6-8189-4c6e-9f10-73de8c91b79f.png)

Reply:
kalau saya mah aliran ini mang @azophy 😄  karena kita base on docker container

<img width="786" alt="image" src="https://user-images.githubusercontent.com/41193120/185584592-c685897d-9142-45f6-86b2-ef2add430d61.png">

jadi new Date() defaultnya akan timezone nya utc +00 sehingga netral timezone nya

Reply:
sebenernya ingin nya di bahas juga di QnA JDS 😄

Reply:
oke2. kalau sudah dipastikan sih aman